develop/tutorials/articles/110-portlets/00-portlets-intro.markdown

```diff
@@ -34,12 +34,13 @@ Specification 1.0
 first defined portal and portlet behavior. In 2008, Java Portlet Specification 
 2.0 
 ([JSR-286](https://jcp.org/en/jsr/detail?id=286)) 
-refined and built on JSR-168, while maintaining backwards compatibility, to 
-define features like inter-portlet communication (IPC) and more. The recently 
-released Java Portlet Specification 3.0 
+refined and built on JSR-168, while maintaining backwards compatibility, to
+define features like inter-portlet communication (IPC) and more. In 2017, Java
+Portlet Specification 3.0 
 ([JSR-362](https://jcp.org/en/jsr/detail?id=362)) 
-continues portal and portlet evolution. Liferay leads in this space by having a 
-member in the Expert Group. 
+continued portal and portlet evolution, bringing features such as configuration
```

Comment:
@jhinkey Recommend "continued to improve the portlet standard" rather than mentioning "portals" since the portlet spec does not really concern itself with portals, but with portlet containers.